src/backends/onnx/onnx_backend.cc

```diff
@@ -301,6 +301,16 @@ OnnxBackend::CreateExecutionContext(
     glock.lock();
   }
 
+  // Register all op libraries that contains custom operations.
+  if (Config().has_model_operations()) {
+    auto model_ops = Config().model_operations();
+    for (const auto& lib_filename : model_ops.op_library_filename()) {
+      void* library_handle = nullptr;  // leak this, no harm.
```

Comment:
Is something actually leaking? What are you supposed to do with this handle?

Reply:
This is not documented. It is from the onnxruntime example [here](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/test/shared_lib/test_inference.cc). From what I understand the op libraries should be unregistered as a part of the cleanup after the session is closed.